CppCoreGuidelines.md

```diff
@@ -6493,13 +6493,6 @@ We can fix that problem by making ownership explicit:
 
 ##### Note
 
-The fact that there are billions of lines of code that violate this rule against owning `T*`s cannot be ignored.
-This code cannot all be rewritten (ever assuming good code transformation software).
-This problem cannot be solved (at scale) by transforming all owning pointer to `unique_ptr`s and `shared_ptr`s, partly because we need/use owning "raw pointers" in the implementation of our fundamental resource handles. For example, most `vector` implementations have one owning pointer and two non-owning pointers.
```

Comment:
The description of "most vector implementations" is dangerously vague, as it conflates memory allocation and object construction, both of which are things that need to be "owned". It would be more accurate to say that "one pair of pointers owns the allocation, and another pair of pointers owns the element lifetimes".